hooks/useMqlQuery.ts

```diff
@@ -151,6 +153,7 @@ function mqlQueryReducer(state: State, action: Action): State {
     case "fetchResultsRunning": {
       const now = Date.now();
       const diff = now - (state.fetchStartTime || 0);
+      console.log("diff", diff);
```

Comment:
Did you intend to leave this print statement in?

Reply:
Nope, just took it out.